contracts/feeManager/FeeManager.sol

```diff
@@ -229,4 +232,25 @@ contract FeeManager is IFeeManager, Ownable {
 
         return funds;
     }
+
+    /**
+     * @notice Overrides the transferOwnership function of Ownable
+     * to remove FINANCIAL_OWNER role from previous _feesOwner
+     * @param newOwner The address of the new owner
+     */
+    // slither-disable-next-line missing-zero-check
+    function transferOwnership(address newOwner) public override {
+        removeFinancialOwner(_feesOwner);
+        _feesOwner = newOwner;
```

Comment:
Slither is already complaining about this and the zero check. `transferOwnership` does that verification, maybe we can either do an explicit check or just move the line to the bottom. Thoughts?